BA OpenShift 7.12 last moment changes (#176)
mramendi/kie-docs: doc-content/enterprise-only/openshift/operator-deploy-basic-proc.adoc

Disable automatic updates if you want to use a custom image for any component of {PRODUCT}.
====
+

> . If you want to use image tags for downloading images, select the *Use Image Tags* box. This setting can be useful if you use a custom registry, or if directed by Red Hat support.

Reviewer: ```suggestion
. If you want to use image tags for downloading images, select the *Use Image Tags* box. This setting is useful if you use a custom registry or if you are directed by Red Hat support.
```

+

> . If you want to disable SSL connections to your deployment, select the *Disable SSL Routes* box. In this case, all routes that are externally exposed use clear-text (HTTP) connections. If this box is not selected, only secure (HTTPS) routes are exposed externally.

Reviewer: ```suggestion
. If you want to disable SSL connections to your deployment, select the *Disable SSL Routes* box. In this case, all routes that are externally exposed use clear-text (HTTP) connections. 
+
NOTE: If this box is not selected, only secure (HTTPS) routes are exposed externally.
```

+
[NOTE]
====

> Do not enable the controller-based startup strategy in an environment with a high-availability {CENTRAL}.

Reviewer: If startupstrategy is code, then please use code formatting Othewise I don't see why you removed the space.

Author: The removed space is after the full stop. The one in "startup strategy" remains here.
